airbyte-integrations/bases/base-java/javabase.sh

```diff
@@ -15,6 +15,9 @@ if [[ $IS_CAPTURE_HEAP_DUMP_ON_ERROR = true ]]; then
       echo "APPLICATION=$APPLICATION"
   fi
 fi
+#30781 - Allocate 32KB for log4j appender buffer to ensure that each line is logged in a single println
+JAVA_OPTS=$JAVA_OPTS" -Dlog4j.encoder.byteBufferSize=32768"
```

Comment:
This control the size of buffer defined in log4j's ConsoleAppender
(See [here](https://logging.apache.org/log4j/log4j-2.17.1/log4j-core/apidocs/src-html/org/apache/logging/log4j/core/appender/AbstractOutputStreamAppender.html) line 47)

This is the base shell command for all java connector.
We could put this JVM options separately on each connector's build.gradle but it's likely to be forgotten.
@davinchia @evantahler @prateekmukhedkar